lib/ReactViews/Map/Navigation/Items/MyLocation.ts

```diff
@@ -14,6 +14,12 @@ import ViewerMode from "../../../../Models/ViewerMode";
 import { GLYPHS } from "../../../../Styled/Icon";
 import StyleTraits from "../../../../Traits/TraitsClasses/StyleTraits";
 import MapNavigationItemController from "../../../../ViewModels/MapNavigation/MapNavigationItemController";
+import {
+  LookAtTraits,
+  IdealZoomTraits
+} from "../../../../Traits/TraitsClasses/MappableTraits";
+import Cartographic from "terriajs-cesium/Source/Core/Cartographic";
+import { sampleTerrain } from "terriajs-cesium";
```

Comment:
@mwu2018 I think this is the source of some of your problems. This should be imported from the correct file within terriajs-cesium. terriajs-cesium does not support importing without file paths.

Reply:
You can track it back through the webpack build error:
```
ERROR in ./node_modules/terriajs-cesium/Source/Widgets/BaseLayerPicker/createDefaultImageryProviderViewModels.js
Module not found: Error: Can't resolve '/home/runner/work/terriajs/terriajs/node_modules/terriajs-cesium/Source/Assets/Textures/NaturalEarthII' in '/home/runner/work/terriajs/terriajs/node_modules/terriajs-cesium/Source/Widgets/BaseLayerPicker'
 @ ./node_modules/terriajs-cesium/Source/Widgets/BaseLayerPicker/createDefaultImageryProviderViewModels.js 237:15-128
 @ ./node_modules/terriajs-cesium/Source/Widgets/Viewer/Viewer.js
 @ ./node_modules/terriajs-cesium/Source/Cesium.js
 @ ./lib/ReactViews/Map/Navigation/Items/MyLocation.ts
 @ ./lib/ReactViews/Map/Navigation/registerMapNavigations.tsx
 @ ./lib/ReactViews/Map/Navigation/MapNavigation.tsx
 @ ./lib/ReactViews/StandardUserInterface/MapColumn.jsx
 @ ./lib/ReactViews/StandardUserInterface/StandardUserInterface.tsx
 @ ./lib/ReactViewModels/ViewState.ts
 @ ./test/Models/MapNavigation/MapNavigationModelSpec.ts
 @ multi ./test/SpecMain.ts ./test/Core/addedByUserSpec.ts ...
 ```
 
 The first line in the "stack trace" that is in terriajs code is MyLocation.ts importing Cesium.js

Reply:
You are spot-on. I used vscode quick-fix for automatically import, which did not raise any warnings. The error only shows up in the CI build.